frontends/web/src/components/icon/icon.tsx

```diff
@@ -80,6 +79,11 @@ export const Info = (props: IconProps) => (<img src={info} draggable={false} {..
 export const BitBox02Stylized = (props: IconProps) => (<img src={BB02Stylized} draggable={false} {...props} />);
 export const ArrowDown = (props: IconProps) => (<img src={arrowDownSVG} draggable={false} {...props} />);
 export const Check = (props: IconProps) => (<img src={checkSVG} draggable={false} {...props} />);
-export const Checked = (props: IconProps) => (<img src={checkedSVG} draggable={false} {...props} />);
 export const Cancel = (props: IconProps) => (<img src={cancelSVG} draggable={false} {...props} />);
 export const Copy = (props: IconProps) => (<img src={copySVG} draggable={false} {...props} />);
+export const Checked = (props: IconProps) => (
+    <svg class="checked" viewBox="0 0 52 52" draggable={false} {...props}>
```

Comment:
Also need to make sure that `props.className` do not overwrite `class="checked"`

Reply:
fixed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/c35550a2a18fb97e23c80966fd047a126bb3068c..16f6593e1dd6920e88a12c47c7ee539299fdbe88